lib/mina/dsl.rb

```diff
@@ -15,6 +15,15 @@ def invoke(task, *args)
       Rake::Task[task].reenable
     end
 
+    def ensure_git_pushed!(opts = {})
+      branch = opts[:branch] || fetch(:branch)
```

Comment:
`fetch` methoda has a second, default argument. so you can write `fetch(:branch, opts[:branch])`